labellab-flask/api/controllers/userscontroller.py

```diff
@@ -0,0 +1,189 @@
+from flask.views import MethodView
+from flask import make_response, request, jsonify, current_app
+from flask_jwt_extended import (
+    create_access_token,
+    create_refresh_token,
+    jwt_required,
+    jwt_refresh_token_required,
+    get_jwt_identity,
+    get_raw_jwt,
+)
+from api.models.User import User
+
+
+class Register(MethodView):
+    """This class registers a new user."""
+
+    def post(self):
+        """Handle POST request for this view. Url --> /api/users/register"""
+        # getting JSON data from request
+        post_data = request.get_json(silent=True,force=True)
+
+        try:
+            name = post_data["name"]
+            username = post_data["username"]
```

Comment:
Can we drop ```username``` attribute and use just the email? @ivantha @Tushar19varshney

Reply:
okay sure we can remove it if others also agree

Reply:
I'll merge this for now. We'll remove it later if it is redundant.